### ISSUE TITLE
Build failed because of lvgl.h  include

### ISSUE THREAD

**Issue by kdschlosser** [kisvegabor/lvgl#6055]
### LVGL version

master branch

### What happened?

```
lvgl/src/font/lv_font_simsun_14_cjk.c:10:14: fatal error: lvgl/lvgl.h: No such file or directory
10 |     #include "lvgl/lvgl.h"
|              ^~~~~~~~~~~~~
```

in the `lv_font_simsun_14_cjk.c` font the `lvgl.h` include is written like this.

```
#ifdef LV_LVGL_H_INCLUDE_SIMPLE
    #include "lvgl.h"
#else
    #include "lvgl/lvgl.h"
#endif
```

in the `lv_font_simsun_16_cjk.c` font the `lvgl.h` include is written like this.

```
#ifdef LV_LVGL_H_INCLUDE_SIMPLE
    #include "lvgl.h"
#else
    #include "../../lvgl.h"
#endif
```

changing the include to read like the bottom one seen above fixes the problem.

**Comment by liamHowatt** [comment]
Thanks @kdschlosser, we're on it. #6050

**Comment by kdschlosser** [comment]
no worries m8

**Comment by kisvegabor** [comment]
Just merged #6050.
